tests/blockchain/eth/contracts/main/staking_escrow/test_staking_escrow_additional.py

```diff
@@ -452,7 +453,7 @@ def test_worker(testerchain, token, escrow_contract, deploy_contract):
     worker_log = escrow.events.WorkerSet.createFilter(fromBlock='latest')
 
     # Initialize escrow contract
-    tx = escrow.functions.initialize().transact({'from': creator})
+    tx = escrow.functions.initialize(0).transact({'from': creator})
```

Comment:
`0`?

Reply:
Just for tests where mining calculation is not necessary